src/main/kotlin/no/nav/medlemskap/routes/CreateDatagrunnlag.kt

```diff
@@ -73,13 +73,24 @@ suspend fun defaultCreateDatagrunnlag(
 }
 
 suspend fun hentDataOmBarn(fnrBarn: List<String>, services: Services, callId: String): List<DataOmBarn> {
-    return fnrBarn.map { DataOmBarn(personhistorikkBarn = hentPersonHistorikkForBarn(it, services, callId)) }
+    val dataOmBarn: MutableList<DataOmBarn> = ArrayList()
```

Comment:
Jeg vil gi deg den samme utfordringen som jeg ga Ole Christian for en stund tilbake - se om du klarer å skrive kode med mer bruk av funksjonell/deklarativ stil, ikke så imperativt som det du har gjort her. Jeg vil f.eks tro at koden du har her kunne vært skrevet som dette:
```
fnrBarn
            .map{ hentPersonHistorikkForBarn(it, services, callId) }
            .filterNotNull()
            .map { dataOmBarn.add(DataOmBarn(it)) }
```
Og IntelliJ vil da tom anbefale deg å forkorte det ned til det følgende:
```
fnrBarn.mapNotNull { hentPersonHistorikkForBarn(it, services, callId) }
            .map { dataOmBarn.add(DataOmBarn(it)) }
```
Jeg synes den typen kode er bedre, men klart, ymmv.. :-)